specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2021-02-10/service.json

```diff
@@ -7230,23 +7988,34 @@
         "tags": [
           "SupportedOperatingSystems"
         ],
-        "summary": "Gets the data of supported OSes by SRS.",
+        "summary": "Gets the data of supported operating systems by SRS.",
         "operationId": "SupportedOperatingSystems_Get",
         "produces": [
           "application/json"
         ],
         "parameters": [
           {
+            "required": true,
             "$ref": "#/parameters/ApiVersion"
           },
           {
+            "required": true,
             "$ref": "#/parameters/ResourceName"
           },
           {
+            "required": true,
             "$ref": "#/parameters/ResourceGroupName"
           },
           {
+            "required": true,
             "$ref": "#/parameters/SubscriptionId"
+          },
+          {
```

Comment:
Please make sure to test this

Reply:
added in test pass document.

---
In reply to: [592111997](https://github.com/AsrOneSdk/azure-rest-api-specs/pull/16#discussion_r592111997) [](ancestors = 592111997)